### ISSUE TITLE
Instagram Ripper only getting first page

### ISSUE THREAD

**Issue by thinkpad4** [RipMeApp/ripme#474]
@cyian-1756 
* Ripme version: 1.7.25
* Java version: Version 8 Build 161
* Operating system: Win 7 64bit
* Exact URL you were trying to rip when the problem occurred:
https://instagram.com/taylorswift  This is a URL I tested it with, but it does it with ALL Instagram URL's 
* Please include any additional information about how to reproduce the problem:
When I start the rip it downloads the first 12 pictures then starts saying "Downloading Next Page"  but there is nothing other than that showing.  
## Expected Behavior
To download all the pictures
Detail the expected behavior here.

## Actual Behavior
[Bug.txt](https://github.com/RipMeApp/ripme/files/1813835/Bug.txt)
That is what it is doing, I've tried several different instagram sites.
Detail the actual (incorrect) behavior here. You can post log snippets or attach log files to your issue report.

**Comment by cyian-1756** [comment]
Does anyone have any idea why this regex won't work? Because I'm completely stumped

```
 Pattern jsP = Pattern.compile("queryId:\"([a-zA-Z0-9]+)\"");
 Matcher m = jsP.matcher("queryId:\"TEST\"");
 return m.group(1);
```

**Comment by thinkpad4** [comment]
Was the \ the issue that bugged it?

**Comment by cyian-1756** [comment]
> Was the \ the issue that bugged it?

No idea, I just replaced the `\"` with a . wildcard

**Comment by cyian-1756** [comment]
This was fixed in 1.7.26

**Comment by thinkpad4** [comment]
@cyian-1756 Thank you SO much for fixing this.  I'm gonna tell you, it seems to rip faster now, like a LOT faster
